src/config/settings.js

```diff
@@ -6,3 +6,33 @@ export const DEFAULT_API_HOST =
   window.parent.location.hostname === 'localhost' ? LOCAL_API_HOST : null;
 
 export const DEFAULT_QUERY = 'education';
+
+export const APP_SETTING_NAMES = {
+  SHOW_QR_CODE: 'showQrCode',
+  SHOW_MODEL: 'showModel',
+  MODEL: 'model',
+};
+
+export const ENABLE_MOCK_API =
+  process.env.REACT_APP_ENABLE_MOCK_API === 'true' ?? false;
```

Comment:
We should decide which version we want to use, to make it uniform accross the apps to not confuse people+devs.